cprover_bindings/src/goto_program/symtab_transformer/identity_transformer.rs

```diff
@@ -247,7 +247,7 @@ mod tests {
 
         let struct_type = Symbol::struct_type(
             "s",
-            None,
+            "s".into(),
```

Comment:
Why is `.into()` needed?

Reply:
Because the function expects an `InternedString`.